.travis.yml

```diff
@@ -0,0 +1,89 @@
+#=============================================================================
+# Travis CoreNeuron settings
+#=============================================================================
+
+#=============================================================================
+# Environment
+#=============================================================================
+# Use new Travis infrastructure (Docker can't sudo yet)
+sudo: false
+language: cpp
+
+#=============================================================================
+# Build matrix
+#=============================================================================
+matrix:
+  fast_finish: true
+  include:
+    - os: linux
+      dist: xenial
+      env:
+        - cmake_option="-DCORENRN_ENABLE_MPI=ON"
```

Comment:
Could you please now also add the builds with the NMODL backend?

Reply:
Additionally, currently you only build with gcc. Could you please also build with clang? For both, could you please choose recent versions? gcc 9.2 and clang 9.0

Reply:
I have rebased my branch on the "cmake" branch to include the sub module NMODL. 
The latest version of the travis CI config contains two more build tests:
1. CORENRN_ENABLE_NMODL=ON 
2. build with linux gcc 9
The source tool chain for Clang 9 is currently not available in Travis. An older version should work, I can add it if needed.